libraries/AP_NavEKF2/AP_NavEKF2.h

```diff
@@ -51,6 +51,11 @@ class NavEKF2 {
         return num_cores;
     }
 
+    // Gets the lane number which is used as primary
+    uint8_t currentPrimary(void) const {
```

Comment:
Could you name this function something like`currentPrimaryLane` or `currentPrimaryLaneId`

Reply:
Done. Thanks!